client/src/components/Icons/GoogleDocIcon.jsx

```diff
@@ -0,0 +1,16 @@
+import GoogleDocIcon from "../../images/GoogleDocsLogo.svg";
+
+
+
+const GoogleDocsLogosvg = () => {
+    return (
+       < a 
+       className="google-doc-project-brief-icon" 
+        href="https://github.com/Gayle-Thompson-Igwebike/GOOD-PR-v1">
```

Comment:
This is good. Will need to do a fetch to get all the urls for the icons

Reply:
Awesome. Thank you Andriana.